tests/Controller/ContentControllerTest.php

```diff
@@ -15,7 +15,7 @@
 
 class ContentControllerTest extends WebTestCase
 {
-    public function test_homepage_works()
+    public function test_homepage_works(): void
```

Comment:
Not a big fan of typing not critical code / test code that never ever ever return anything (as it's a test).

Reply:
Not a big fan either but I think it do not harm us to be forced to write typed PHP :grin: